templates/resources/vmaas_load_balancer_profile.md.tmpl

```diff
@@ -0,0 +1,52 @@
+---
+layout: ""
+page_title: "hpegl_vmaas_load_balancer_profile Resource - vmaas-terraform-resources"
+subcategory: {{ $arr := split .Name "_" }}"{{ index $arr 1 }}"
+description: |-
+  {{ .Description | plainmarkdown | trimspace | prefixlines "  " }}
+---
+
+# Resource hpegl_vmaas_load_balancer_profile
+
+-> Compatible version >= 5.4.6
+
+{{ .Description | trimspace }}
+`hpegl_vmaas_load_balancer_profile` resource supports NSX-T Load balancer Profile creation.
+
+For creating an NSX-T Load balancer Profile, use the following examples.
+
+-> NSX-T Profile having HTTP, UDP, TCP, COOKIE, SOURCEIP, GENERIC, SERVER, CLIENT considered as different Profile types. You can create either any of them HTTP, HTTPS, ICMP, Passive, TCP, UDP at a given time
```

Comment:
NSX-T Profile having HTTP, HTTPS, UDP, TCP, COOKIE, SOURCEIP, GENERIC, SERVER and CLIENT considered as different Profile types. You can create any one of them at a given time

this statement is better.